enablesanitizers.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
Isn't enabling sanitizers a build process thing? What is the purpose of setting env variables and then consuming them in the build instead of adding these options to the build.sh itself?